src/rdkafka_msgset_reader.c

```diff
@@ -208,7 +208,8 @@ typedef struct rd_kafka_msgset_reader_s {
 static rd_kafka_resp_err_t
 rd_kafka_msgset_reader_run (rd_kafka_msgset_reader_t *msetr);
 static rd_kafka_resp_err_t
-rd_kafka_msgset_reader_msgs_v2 (rd_kafka_msgset_reader_t *msetr);
+rd_kafka_msgset_read_msgs_v2 (rd_kafka_msgset_reader_t *msetr,
```

Comment:
rd_kafka_msgset_reader_ is the prefix for all of these functions, think of it as a class.

Reply:
what about `rd_kafka_msgset_reader_read_msgs_v2`?

Reply:
We dont need to rename it.

Reply:
ok, just trying to improve readability, the suggested name would have helped me.